routes/index.js

```diff
@@ -7,22 +7,24 @@ var routes = {
   'signup': function(request, reply) {
     var url = process.env.SIGNUP;
     var transaction = request.payload || {};
-    if(transaction.locale !== 'en-US') {
-      url = url + '-' + transaction.locale;
-    }
-    httpRequest({
+    httpRequest.post({
       url: url,
-      method: 'POST',
       form: {
-        'opt_in': '1',
+        format: 'html',
+        lang: transaction.locale,
```

Comment:
This is fine, so long as we're only showing the sign-up thank you page for `en` and `de` donors. We haven't setup mailing list for other locales.